Chapters/FirstContact/FirstContactShrunk.md

```diff
@@ -10,7 +10,7 @@ After completing this chapter you should read Chapter *@cha_reuse@* about reuse
 
 ![A screen shot of the customer satisfaction UI.](figures/CustomersBasic.png width=50&label=figCustomersBasic)
 
-We construct a simple customer satisfaction UI, which allows a user to give feedback about a service by clicking on one of three buttons. \(This feedback should be recorded and processed, but this is outside of the scope of this example\). We show a screenshot of the UI in Figure *@figCustomersBasic@*.
+We construct a simple customer satisfaction UI, which allows a user to give feedback about a service by clicking on one of three buttons. This feedback should be recorded and processed, but this is outside of the scope of this example. We show a screenshot of the UI in Figure *@figCustomersBasic@*.
```

Comment:
I changed this because there is no need to put the sentence between parentheses.

Reply:
Yes this was a mistake in the pillar to MD and I did not fix it.